htdocs/takepos/invoice.php

```diff
@@ -195,6 +195,8 @@ function fail($message)
 	//	else $conf->global->FACTURE_ADDON = $conf->global->TAKEPOS_ADDON;
 	//}
 
+	if ($pay == "invoice") $invoice->module_source = '';
```

Comment:
Why removing this ?
This is a tracking information to know that the invoice was generated from POS instead of from backoffice.